circle.yml

```diff
@@ -40,7 +39,10 @@ machine:
 
 checkout:
   post:
-    - git clone --depth 1 -b ${ST2_PACKAGES_BRANCH} ${ST2_PACKAGES_REPO} /home/ubuntu/st2/st2-packages
+    - |
+      git clone --depth 1 ${ST2_PACKAGES_REPO} /home/ubuntu/st2/st2-packages
+      cd /home/ubuntu/st2/st2-packages
+      git checkout ${CIRCLE_BRANCH} || true
```

Comment:
I'm still hoping someday we would just merfe st2 and st2-packages and forget about this problem once and for all.